src/generator/download.ts

```diff
@@ -78,7 +80,7 @@ export async function downloadDiscoveryDocs(
         options.downloadPath,
         api.id.replace(':', '-') + '.json'
       );
-      const url = api.discoveryRestUrl;
+      const url = `${options.discoveryUrl}/${api.id.replace(':', '.')}.json`;
```

Comment:
I would use api.name + api.version as that is what the discovery cache updater uses to build the filename. I can't guarantee that api.id is the same.